app/src/main/java/knu/dong/teamproject/MyPageActivity.kt

```diff
@@ -0,0 +1,53 @@
+package knu.dong.teamproject
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import androidx.core.content.edit
+import knu.dong.teamproject.common.HttpRequestHelper
+import knu.dong.teamproject.databinding.ActivityMyPageBinding
+import knu.dong.teamproject.dto.UserInfoDto
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+class MyPageActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val binding = ActivityMyPageBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+
+        binding.titleBar.btnBack.setOnClickListener {
+            onBackPressed()
+        }
+        binding.titleBar.title.text = "유저 페이지"
+        binding.titleBar.btnAccount.visibility = View.INVISIBLE
+        binding.btnLogout.setOnClickListener {
+            CoroutineScope(Dispatchers.Main).launch {
+                logout()
+            }
+        }
+        val sharedUserInfo = getSharedPreferences("user_info", MODE_PRIVATE)
+
+        val userId = sharedUserInfo.getLong("id", -1)
+        CoroutineScope(Dispatchers.Main).launch {
+            var userInfo = HttpRequestHelper(this@MyPageActivity).get("api/users", UserInfoDto::class.java)
+            binding.userEmail.text = userInfo?.email
+            binding.rateLimit.text = "${userInfo?.currentUsedCount} / ${userInfo?.rateLimit}"
+        }
+    }
+
+    private suspend fun logout() {
+        HttpRequestHelper(this).post("api/users/logout")
+
+        getSharedPreferences("user_info", MODE_PRIVATE).edit {
+            remove("id")
+        }
+
+        val intent: Intent = Intent(this, LoginActivity::class.java)
+        startActivity(intent)
+    }
```

Comment:
```suggestion
    private suspend fun logout() {
        HttpRequestHelper(this).post("api/users/logout")

        getSharedPreferences("user_info", MODE_PRIVATE).edit {
            remove("id")
        }

        val intent: Intent = Intent(this, LoginActivity::class.java)
        startActivity(intent)
        
        ActivityCompat.finishAffinity(this)
    }
```

`startActivity`를 하고 `finish`를 해주지 않으면, 스택에 기존 액티비티가 살아있는 채로 남아있음. 즉, 뒤로가기를 하면 다시 해당 액티비티 페이지로 돌아옴. 로그아웃을 했기에 로그인 페이지에서 뒤로가기를 했다고 마이페이지로 돌아가면 안됨.
단, `finish`는 호출한 액티비티만 스택에서 제거해줌. 마이페이지는 챗봇 페이지 -> 마이페이지 혹은 챗봇 -> 채팅 -> 마이페이지로 가기 때문에, `finish`만 호출하면 이전 챗봇 페이지와 채팅 페이지는 스택에 남아있게 됨. 그렇기에 모든 액티비티를 스택에서 제거하기 위해 `ActivityCompat.finishAffinity(this)` 함수를 사용함.

Reply:
해당 부분은 내가 수정할겡